README.md

```diff
@@ -16,11 +16,9 @@ For more information about how the protocols work in this scenario and other sce
 
 To run this sample you will need:
 - Visual Studio 2013
-- The Azure AD module for Powershell
 - The makecert.exe utility from the Windows SDK
 - An Internet connection
 - An Azure Active Directory (Azure AD) tenant. For more information on how to get an Azure AD tenant, please see [How to get an Azure AD tenant](https://azure.microsoft.com/en-us/documentation/articles/active-directory-howto-tenant/) 
-- A user account in your Azure AD tenant. This sample will not work with a Microsoft account, so if you signed in to the Azure portal with a Microsoft account and have never created a user account in your directory before, you need to do that now.
 
 ### Step 1:  Clone or download this repository

```

Comment:
Visual Studio is pretty good at cloning repositories, no need to go to the shell